tests/testthat/test-derive_vars_aage.R

```diff
@@ -51,3 +51,89 @@ test_that("derive_var_age_years - Error is issued if age_unit is missing", {
     derive_var_age_years(input, AGE, new_var = AAGE)
   )
 })
+
+test_that(paste0("derive_var_age_years - Warning is issued if age_unit is not null, but the ",
```

Comment:
I think you can just put these on new lines and remove the paste0.  Also you can use  `#nolint` to the end of the line and the lintr won't checkt hat line.  Don't use it here, but just FYI

Reply:
Can you number these tests following the guidance here.  There is an addin in `admiraldev` that can do this for you. 

https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#set-up-the-test-script

https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file

Reply:
> I think you can just put these on new lines and remove the paste0. Also you can use `#nolint` to the end of the line and the lintr won't checkt hat line. Don't use it here, but just FYI

When I remove `paste0()`, then I can only specify a single text string: I can't specify `"abc" "def"`, nor can I specify `"abc", "def"`; I can only specify `"abc _<new line>_ def"`. If I then force the testthat script to fail, then the message will be split wherever I have a new line in the testthat script:

`Failure (test-derive_vars_aage.R:68:3): derive_var_age_years - Warning is issued if age_unit is not null, but the 'unit' variable corresponding to age_var stores more than one unique value.`
 
, becomes

```
Failure (test-derive_vars_aage.R:67:3): derive_var_age_years - Warning is issued if age_unit is not null, but the 'unit'
           variable corresponding to age_var stores more than one unique value.

```
So I think the messages are neater when I used paste0(). If there is another way to split the text in the testthat script without resulting in unnecessary newlines in the actual messages, then please let me know.
BTW, the above example doesn't look so bad when I copy and paste into GitHub, but in the R Studio console, the text `unit` appears near the start of a new line, so it looks odd when `variable corresponding to` starts on the next line.

Reply:
As discussed during a team meeting, I've removed the calls to paste0() when I call test_that().